jenkins/scripts/integration_test.sh

```diff
@@ -174,8 +174,7 @@ echo "Running the tests"
 ssh \
   -o StrictHostKeyChecking=no \
   -o UserKnownHostsFile=/dev/null \
-  -o ServerAliveInterval=15 \
-  -o ServerAliveCountMax=10 \
+  -o ServerAliveInterval=10800 \
```

Comment:
This is 3h ? does it mean that the server will try to reconnect for 3h after broken pipe

Reply:
No, this means that the server waits for 3 hours after the last char was sent on ssh.

Reply:
Broken pipe can happen simply because the connection times out because there is no communication on the connection. 
This situation can easily happen if the test takes a lot of time and it does not print anything to the console.